src/main/java/io/specto/hoverfly/junit/dsl/matchers/PlainTextMatcher.java

```diff
@@ -0,0 +1,6 @@
+package io.specto.hoverfly.junit.dsl.matchers;
+
+public interface PlainTextMatcher extends RequestMatcher {
```

Comment:
It is a bit similar to marker interface. Exact/glob/regex matcher will implement this subset of RequestMatcher, so body field can support all RequestMatchers, while url, path, query, method only supports PlainTextMatcher. It just provides type information.